README.rst

```diff
@@ -105,6 +102,22 @@ from the github repository or archive, unpack, and run from within the
 toplevel `python-control` directory::
 
   pip install .
+  
+Article and Citation Information
+================================
+
+A `2021 article <https://ieeexplore.ieee.org/abstract/document/9683368>`_ about the library is available on IEEE Explore. If the Python Control Systems Library helped you in your research, please cite::
+
+  @inproceedings{python-control2021,
+    title={The python control systems library (python-control)},
```

Comment:
I think we should capitalize Python or capitalize the entire title (which is how it appears on [IEEE Explore](https://ieeexplore.ieee.org/document/9683368): "The Python Control Systems Library (python-control)".